src/renderers/shaders/ShaderChunk/output_fragment.glsl.js

```diff
@@ -5,8 +5,8 @@ diffuseColor.a = 1.0;
 
 // https://github.com/mrdoob/three.js/pull/22425
 #ifdef USE_TRANSMISSION
-diffuseColor.a *= material.transmissionAlpha + 0.1;
+diffuseColor.a *= material.transmissionAlpha;
 #endif
 
-gl_FragColor = vec4( outgoingLight, diffuseColor.a );
+gl_FragColor = vec4( outgoingLight * diffuseColor.a, diffuseColor.a );
```

Comment:
In this update I no longer need any changes to the default renderer 🎉, instead applying our `premultipliedAlpha` here. The results look better too - the colors stay more saturated. The only issue I notice is a dark outline due to interpolating toward transparent black: 
<img width="218" alt="image" src="https://user-images.githubusercontent.com/1649964/231614137-75389d98-54d3-48d8-9260-d411ad823469.png">
Not too bad in the grand scheme of things. However, this also made some regression render diffs, so we'll need to look at that.

Reply:
`output_fragment` is used by almost every built-in material. You can't just multiply RGB by alpha. You will break every example using transparency.

This is why we have the property `material.premultipliedAlpha`.

Reply:
Thanks, I switched to enabling `premultipliedAlpha` instead. This got me back to the first draft renders, so if I'd tried this first I probably would have been totally happy with the result. But now I have that 2nd draft render to compare to with more saturation that I like better. I can't quite figure out what's causing the difference.